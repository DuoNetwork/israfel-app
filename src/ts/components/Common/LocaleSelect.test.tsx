/**
 * @jest-environment jsdom
 */

import { shallow } from 'enzyme';
import * as React from 'react';
import LocaleSelect from './LocaleSelect';

describe('LocaleSelect Test', () => {
	const locale = 'EN';
	const onSelect = jest.fn();

	describe('Test Snapshot', () => {
		it('Test Snapshot', () => {
			const wrapper = shallow(< LocaleSelect locale={locale} onSelect={onSelect} />);
			expect(wrapper).toMatchSnapshot();
		});
	});
});
