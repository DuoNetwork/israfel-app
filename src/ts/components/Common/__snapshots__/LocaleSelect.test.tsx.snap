// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LocaleSelect Test Test Snapshot Test Snapshot 1`] = `
<div>
  123
</div>
`;
